source/jormungandr/jormungandr/travelers_profile.py

```diff
@@ -79,7 +79,8 @@ def override_params(self, args):
                               ('destination_mode',           self.last_section_mode),
                               ('wheelchair',                 self.wheelchair))
 
-        def override((arg, profile_attr)):
+        def override(tupe):
```

Comment:
`tuple` is a "reserved" word, I can override it but I don't really like it, but I can rename it `pair` it will be better